sal_interfaces/SALSubsystems.xml

```diff
@@ -587,8 +587,8 @@
   <SoftwareLanguage>Python</SoftwareLanguage>
   <RuntimeLanguages>IDL</RuntimeLanguages>
   <VendorContact>Not Applicable</VendorContact>
-  <Simulator>No</Simulator>
-  <Configuration>Not Configurable</Configuration>
+  <Simulator>Internal to CSC</Simulator>
+  <Configuration>Not Yet Configurable</Configuration>
```

Comment:
This needs to be a guess at a URL. Making one up if there is none is OK for now. If ESS is part of the EAS, I would use that GH URL.

Reply:
The CSC for this is not configurable yet, which is why I set it to "Not yet". I changed it to "Not Configurable".